lua/gk.lua

```diff
@@ -0,0 +1,21 @@
+local ffi = require("ffi")
+local gk_config = require("gk_conf")
+local gatekeeperc = require("gatekeeperc")
+
+local M = {}
+
+-- Function that sets up the GK functional block.
+function M.setup_block()
+	local ret = 0
+
+	-- Init the GK configuration structure.
+	conf = gatekeeperc.alloc_and_return_gk_conf()
+	conf.lcore_start_id = gk_config.lcores.lcore_start_id
```

Comment:
Lua scripts are not compiled as the C code, so the same effort that one has to edit a table variable is not much different of commenting code out. The idea is for the configuration file to easily be changed and take advantage of copy and paste.